packages/components/src/menu/context.tsx

```diff
@@ -6,8 +6,6 @@ import { createContext } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import type { MenuContext as MenuContextType } from './types';
+import type { ContextProps } from './types';
 
-export const MenuContext = createContext< MenuContextType | undefined >(
-	undefined
-);
+export const Context = createContext< ContextProps | undefined >( undefined );
```

Comment:
I feel like those renames should have happened in a separate PR, but no big deal I guess.

Reply:
Agreed, and agreed.